### PR TITLE
Add modal dismissal

### PR DESCRIPTION
**Pull request by dj-pundir**
This is the resolution of issue https://github.com/ChainSafe/Multix/issues/132

Followings are the changes:
- Add close icon for dialog dismissal.
- Prevent dialog dismiss on user outside click.
- Use Close Icon button from the existing page "NewMulisigAlert"
- Add dismissal on all dialogs existing currently in the application.
- Run the lint --fix command.

Here is the popup dialog now:

image

### CODE REVIEW

**Comment by asnaith** [comment]
Just an FYI @Tbaut, the CloudFlare deploy to preview failed here due to an api key issue

**Comment by asnaith** [comment]
Thank you for the contribution @dj-pundir! This is working nicely. 

I'll wait for @Tbaut to approve as I think he will want to review the code before it's merged and right now we're only requiring one approval.

**Comment by Tbaut** [comment]
Yes cloudflare deployment doesn't like external contributions unfortunately. 
Sorry for the long wait for review, I've been on some big PRs lately. Will take a closer look tomorrow.

File: packages/ui/src/components/modals/ChangeMultisig.tsx
Comment (Tbaut) [inline packages/ui/src/components/modals/ChangeMultisig.tsx:369]: I suggest to place it as the first element, outside of the `DialogTitle` and also no need to check for the `onClose`, this will never be undefined.

```suggestion
        <IconButton
          className="closeButton"
          size="small"
          aria-label="close"
          color="inherit"
          onClick={onClose}
        >
          <CloseIcon fontSize="small" />
        </IconButton>
      <DialogTitle>Change multisig</DialogTitle>
```

**Review by Tbaut — CHANGES_REQUESTED** [review]
Thanks a lot @dj-pundir for your contribution. I suggested a little change (please change it for all the occurences). The rest looks good.

File: packages/ui/src/components/modals/ChangeMultisig.tsx
Comment (dj-pundir) [reply packages/ui/src/components/modals/ChangeMultisig.tsx]: Done, Thanks!!

File: packages/ui/src/components/modals/ChangeMultisig.tsx
Comment (Tbaut) [inline packages/ui/src/components/modals/ChangeMultisig.tsx:378]: Add the check on `isCallStep` to keep the previous behavior
```suggestion
      {!isCallStep && (
        <IconButton
          className="closeButton"
          size="small"
          aria-label="close"
          color="inherit"
          onClick={onClose}
        >
          <CloseIcon fontSize="small" />
        </IconButton>
      )}
```

**Review by Tbaut — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/modals/EditNames.tsx
Comment (Lykhoyda) [inline packages/ui/src/components/modals/EditNames.tsx:146]: We are getting rid of classes in our code. Please use the styled component to apply the styles (https://emotion.sh/docs/styled). As you are using the same icon and styles 3 times, you might create a component for that, f.e in src/components/library/CloseButton.tsx

**Review by Tbaut — APPROVED** [review]
Thanks a lot @dj-pundir I think this is good to go.

**Pull request merged**
